code/game/objects/items/devices/radio/radio.dm

```diff
@@ -247,14 +247,12 @@ var/global/list/default_medbay_channels = list(
 	if (!connection)
 		return
 
-	var/mob/living/silicon/ai/A = new /mob/living/silicon/ai(src, null, null, 1)
+	var/static/mob/living/silicon/ai/announcer/A = new /mob/living/silicon/ai/announcer(src, null, null, 1)

```

Comment:
And the `static` here means the Singleton AI will be found when the proc is called by other objects, or will `new` still make fresh copies?

Reply:
The way it appears to work (I was not aware of this before working on this issue, actually) is that declaring a static inside a proc makes it a sort of 'global static' that is only initialized once. I believe it makes it literally a global var, but somehow scoped to just this proc. This line is basically 'skipped' after it's initialized once (in truth, it's initialized when the game starts, this line 'isn't really here). It's a little weird how Byond handles this, but, it works and it's apparently intended to behave like I've described.

Reply:
Ok. Byond reference docs don't describe static, or if they do it's somewhere hidden, but it's not surprising there's support for a Singleton-style object management

Reply:
Oh, sorry. `static` is an alias for `global`, maybe I misunderstood your question.
They are the same thing.